### DISCUSSION TITLE
Error when connecting an accdb (MS Access) file with pyodbc in Google Colab

### DISCUSSION THREAD

**Discussion by alexcatalancarrasco** [mkleehammer/pyodbc#1357] [Q&A] [answered]
Good evening colleagues 👋!

Please, I will appreciate 🙏 your enormous help and guidance with this problem:

When trying to make a connection to an Access file (accdb format), previously loaded in Google Colab, with pyodbc it gives me the following error ⚡:

`Error: ('01000', "[01000] [unixODBC][Driver Manager]Can't open lib 'Microsoft Access Driver (.mdb, .accdb)' : file not found (0) (SQLDriverConnect)")`

- Installed libraries: python 3.10, pyodbc 5.1.0 and pandas 2.0.3
- Driver name: DRIVER_NAME = "Microsoft Access Driver (.mdb, .accdb)"
- File path in Google Colab: DB_PATH = r'\content\dataset.accdb'
- Establish the connection: conn = pyodbc.connect("Driver={%s};DBQ=%s;" % (DRIVER_NAME, DB_PATH))

In advance, any comments with any ideas or guidance to be able to make the connection from Google Colab will be welcome 🙌.

Note: From VSCode with local file it works 100%.

**Comment by gordthompson** [comment, marked as answer]
`Microsoft Access Driver (*.mdb, *.accdb)` is the name of the Microsoft Access ("ACE") driver on Windows. The unixODBC error indicates that it came from a non-Windows machine, but Microsoft has not released a corresponding MS Access driver for Linux or Mac.

You may find this interesting: https://stackoverflow.com/q/25596737/2144390
